site/TopicTags.scss

```diff
@@ -2,16 +2,24 @@
     margin-bottom: 16px;
     @include overline-black-caps;
     color: $blue-50;
+
+    @include sm-only {
+        font-size: 0.625rem;
+        margin-bottom: 8px;
+    }
 }
 
 .topic-tags {
     @include gap-wrap(12px, 16px);
 }
 
 .topic-tag {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
Is this doing anything? toggling the rules doesn't seem to make a change that I can see

Reply:
You are right, removing. It's a leftover from a previous WIP version.